lm75bd/lm75bd.c

```diff
@@ -27,8 +30,21 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
 }
 
 error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {
-  /* Implement this driver function */
-  
+  if (temp == NULL) return ERR_CODE_INVALID_ARG;
+
+  uint8_t tmpPointerRegister = LM75BD_TEMP_REG;
+  i2cSendTo(devAddr, &tmpPointerRegister, 1);
+  uint8_t buffer[2] = {0};
+  i2cReceiveFrom(devAddr, buffer, 2);
```

Comment:
Check the return value of these i2c functions. If a non-success error code is returned, then return that error code.